sdk/python/arvados/arvfile.py

```diff
@@ -516,7 +517,7 @@ def _block_prefetch_worker(self):
                     return
                 self._keep.get(b)
             except Exception:
-                pass
+                _logger.error(traceback.format_exc())
```

Comment:
I agree that knowing that a failure has occurred is a step in the right direction. However, that solution doesn't provide any insight into the nature and origin of the related fault. In addition, it obfuscates the location of that exception handler, making it difficult for someone not familiar with the codebase to know where to start.

A traceback provides much more useful information, including the location you suggest would be useful (albeit expressed without knowledge of the higher level context).